wp-content/mu-plugins/pantheon/pantheon-try-gutenberg-mods.php

```diff
@@ -0,0 +1,62 @@
+<?php
+/**
+  * Should we proceed with adding the
+  * return to Pantheon button?
+  *
+  * Only if:
+  * We are on a Pantheon subdomain and
+  * "RETURN_TO_PANTHEON_BUTTON" is not false
+  */
+$mod_try_gutenberg = apply_filters( 'show_pantheon_try_gutenberg_mods', true);
```

Comment:
@ari-gold the `show_pantheon_try_gutenberg_mods` can be set to `false` to disable our modifications

Reply:
Excellent! 

So you'd add the folllowing line to wp-config,php or somewhere else?

```
$mod_try_gutenberg = apply_filters( 'show_pantheon_try_gutenberg_mods', false);
```